base_report_docx/report/report_docx.py

```diff
@@ -64,11 +64,19 @@ def create_source_docx(self, cr, uid, ids, context=None):
 
         report = self._get_convert_file(
             tmp_folder_name, output_report[output_type])
-
+        # Put the pdf to sharefolder before delete the temp folder
+        self.save_pdf_to_share(cr, uid, ids, tmp_folder_name, context)
```

Comment:
change to `on_delete_output_file(file)`

where `file = tmp_folder_name + output_report[output_type]`

and change the signature of: `_get_convert_file(file)`

Reply:
Also, adapt the comment to explain that this is a hook that can be used to copy the file to the HD (definitely do not mention shared folder, the shared folder is a deployment topic)